RELEASE-NOTES.txt

```diff
@@ -1,6 +1,6 @@
 4.37
 -----
-
+-   Update in-app notifications color for better readability in dark mode #1288
```

Comment:
This is unrelated to the release process, but will make it easier to run the code freeze on Monday.

Otherwise, there would have been no release notes, and we'd have to go find something (this text) to put into them in the middle of the code freeze.